### PR TITLE
[Keyboard] Add NuPhy Krypton75 keyboard

### PR DESCRIPTION
**Pull request by Persama**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/theone/nuphy_krypton75/nuphy_krypton75.c [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+    }
+    switch (keycode) {
+        case MAC_TASK: {
+            if (record->event.pressed) {
+                host_consumer_send(0x029F);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;
+        }
```
Comment (zvecr) [inline keyboards/theone/nuphy_krypton75/nuphy_krypton75.c:491]: This is just `KC_MISSION_CONTROL` but with more steps.

File: keyboards/theone/nuphy_krypton75/nuphy_krypton75.c [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+        case MAC_DNT: {
+            if (record->event.pressed) {
+                host_consumer_send(0x02A0);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;
+        }
```
Comment (zvecr) [inline keyboards/theone/nuphy_krypton75/nuphy_krypton75.c:515]: As is this, `KC_LAUNCHPAD`

File: keyboards/theone/nuphy_krypton75/nuphy_krypton75.c (hunk 1 above)
Comment (Persama) [reply keyboards/theone/nuphy_krypton75/nuphy_krypton75.c]: Thank you, I have adjusted it

File: keyboards/theone/nuphy_krypton75/nuphy_krypton75.c (hunk 2 above)
Comment (Persama) [reply keyboards/theone/nuphy_krypton75/nuphy_krypton75.c]: Thank you, I have adjusted it

**Comment by cassiomolin** [comment]
When I first saw this pull request, I was hoping this would be for the Air75 v2. The pull request mentions the processor `STM32F072`, which is used by the Air75 v2.

But it turns out the number of keys (81) doesn't match the number of keys on the Air75 v2 (84). And the layout is also different.

**Pull request closed without merging**